utils/coreutils_test.go

```diff
@@ -1219,35 +1220,35 @@ Date:   Fri Dec 30 19:48:09 2016 +0100
 	GitLastLog = ""
 	if vers, err := GetCGRVersion(); err != nil {
 		t.Error(err)
-	} else if vers != "CGRateS@v0.11.0~dev" {
-		t.Errorf("Expecting: <CGRateS@v0.11.0~dev>, received: <%s>", vers)
+	} else if vers != version {
```

Comment:
Same story as for the other branch. Please rename expected variable to something more sane.